nemo/collections/asr/models/configs/quartznet_config.py

```diff
@@ -0,0 +1,308 @@
+        JasperEncoderConfig(filters=768, repeat=5, kernel=[25], stride=[1], dilation=[1], dropout=0.3,
+                            residual=True, groups=1, separable=False, heads=-1, residual_mode='add',
+                            residual_dense=True, se=False, se_reduction_ratio=8, se_context_size=-1,
+                            se_interpolation_mode='nearest', kernel_size_factor=1.0, stride_last=False),
+        JasperEncoderConfig(filters=768, repeat=5, kernel=[25], stride=[1], dilation=[1], dropout=0.3,
+                            residual=True, groups=1, separable=False, heads=-1, residual_mode='add',
+                            residual_dense=True, se=False, se_reduction_ratio=8, se_context_size=-1,
+                            se_interpolation_mode='nearest', kernel_size_factor=1.0, stride_last=False),
+        JasperEncoderConfig(filters=896, repeat=1, kernel=[29], stride=[1], dilation=[2], dropout=0.4,
+                            residual=False, groups=1, separable=False, heads=-1, residual_mode='add',
+                            residual_dense=False, se=False, se_reduction_ratio=8, se_context_size=-1,
+                            se_interpolation_mode='nearest', kernel_size_factor=1.0, stride_last=False),
+        JasperEncoderConfig(filters=1024, repeat=1, kernel=[1], stride=[1], dilation=[1], dropout=0.4,
+                            residual=False, groups=1, separable=False, heads=-1, residual_mode='add',
+                            residual_dense=False, se=False, se_reduction_ratio=8, se_context_size=-1,
+                            se_interpolation_mode='nearest', kernel_size_factor=1.0, stride_last=False)
+    ]
+    return config
+# fmt: on
+
+
+@dataclass
+class QuartzNetModelConfig(ctc_cfg.EncDecCTCConfig):
+    # Model global arguments
+    sample_rate: int = 16000
+    repeat: int = 1
+    dropout: float = 0.0
+    separable: bool = True
+    labels: List[str] = MISSING
+
+    # Dataset configs
+    train_ds: ctc_cfg.EncDecCTCDatasetConfig = ctc_cfg.EncDecCTCDatasetConfig(
+        manifest_filepath=None, shuffle=True, trim_silence=True
+    )
+    validation_ds: ctc_cfg.EncDecCTCDatasetConfig = ctc_cfg.EncDecCTCDatasetConfig(
+        manifest_filepath=None, shuffle=False
+    )
+    test_ds: ctc_cfg.EncDecCTCDatasetConfig = ctc_cfg.EncDecCTCDatasetConfig(manifest_filepath=None, shuffle=False)
+
+    # Optimizer / Scheduler config
+    optim: Optional[model_cfg.OptimConfig] = model_cfg.OptimConfig(sched=model_cfg.SchedConfig())
+
+    # Model general component configs
+    preprocessor: AudioToMelSpectrogramPreprocessorConfig = AudioToMelSpectrogramPreprocessorConfig()
+    spec_augment: Optional[SpectrogramAugmentationConfig] = SpectrogramAugmentationConfig()
+    encoder: ConvASREncoderConfig = ConvASREncoderConfig(activation="relu")
+    decoder: ConvASRDecoderConfig = ConvASRDecoderConfig()
+
+
+@dataclass
+class JasperModelConfig(QuartzNetModelConfig):
```

Comment:
small nitpick. Jasper appeared first, so it would be better to have QuartzNetModelConfig inherit JasperModelConfig and set separable=True

Reply:
Done